src/porepy/params/parameter_dictionaries.py

```diff
@@ -0,0 +1,116 @@
+""" Parameter dictionaries.
+
+Here, we store various parameter dictionaries with "sensible" default values for the
+parameters required by the discretization objects.
+Note that the values of the output dictionary are set according to the relationship
+between the physical process and the equation. Thus, the mass_weight ("mathematical
+term") of the flow dictionary is set to the value of the porosity ("pysical term").
+The same goes for the tensors.
+"""
+
+import numpy as np
+import porepy as pp
+
+
+def flow_dictionary(g, in_data={}):
+    """ Dictionary with parameters for standard flow problems.
+
+    All parameters listed below which are not specified in in_data are assigned unitary
+    values. Additional parameters may be passed in in_data.
+    Parameters:
```

Comment:
Do you mean to include a list of the parameters for which there are default values? I'm not sure that I agree, since setting these values is basically the only task of this function, and it should be pretty obvious which keys are filled. I don't have strong feelings on this, however.